scripts/package-distribution.ps1

```diff
@@ -82,8 +82,7 @@ foreach ($Build in $AllBuilds) {
     } else {
         $Binaries += Join-Path $ArtifactsDir "libmsquic.so"
         #Temporary until we fix CLOG
-        $Binaries += Join-Path $ArtifactsDir "libPLATFORM_CLOG_LIB.so"
-        $Binaries += Join-Path $ArtifactsDir "libCORE_CLOG_LIB.so"
+        $Binaries += Join-Path $ArtifactsDir "libmsquictraceprovider.so"
```

Comment:
Can this be named `libmsquic.traceprovider.so` or (since I think it's only necessary when we use lttng) `libmsquic.lttng.so`?

Reply:
Yeah I'll do that.